src/style.scss

```diff
@@ -17,10 +17,10 @@
 // ---------------------------------------------------------------------
```

Comment:
Background color of bordered inputs is now transparent, should be original color